flowmachine/flowmachine/core/server/action_handlers.py

```diff
@@ -124,9 +125,15 @@ def action_handler__run_query(**action_params: dict) -> ZMQReply:
     try:
         query_id = q_info_lookup.get_query_id(action_params)
     except QueryInfoLookupError:
-        # Set the query running (it's safe to call this even if the query was set running before)
+        store_dependencies = not (
```

Comment:
Hmm. Not sure about reading the env var in the handler. I think I'd prefer it was happening in the main server and passed to the handler as an arg, or the handler was selected based on the env var?

Reply:
I've moved all the environment-variable-reading in the server into `server_config.get_server_config`, which is called by `server.main` and returns a `NamedTuple` that can be passed around to the other server functions and action handlers.